drivers/libusb.c

```diff
@@ -552,7 +540,7 @@ static int libusb_get_interrupt(usb_dev_handle *udev, unsigned char *buf, int bu
 	}
 
 	/* FIXME: hardcoded interrupt EP => need to get EP descr for IF descr */
-	ret = usb_interrupt_read(udev, 0x81, (char *)buf, bufsize, timeout);
+	ret = usb_interrupt_read(udev, USB_ENDPOINT_IN + usb_subdriver.hid_ep_in, (char *)buf, bufsize, timeout);
```

Comment:
In the PR changes, I did not see any definition of `USB_ENDPOINT_IN` as `0x81` (or near that, to shift by a small `hid_ep_in` value). Just worrying if this change is correct.